mkt/websites/utils.py

```diff
@@ -27,6 +28,7 @@ def website_factory(**kwargs):
         'title': 'Title for %s' % text.capitalize(),
         'url': 'http://%s.example.com' % text,
         'mobile_url': 'http://mobile.%s.example.com' % text,
+        'devices': DEVICE_TYPES.keys(),
```

Comment:
Yeah, could not decide, and it's just a default - the caller can override any of those parameters, so it doesn't matter much really.